res/schema.xml

```diff
@@ -489,4 +489,29 @@ METADATA
       ALTER TABLE library ADD COLUMN color INTEGER;
     </sql>
   </revision>
+  <revision version="32" min_compatible="3">
+    <description>
+      Convert the PredefinedColor ID to the actual RGB value.
+    </description>
+    <sql>
+      <!-- No Color becomes white-->
+      UPDATE cues SET color=0xFFF2F2FF WHERE color=0;
```

Comment:
We cannot use white here, because alpha users may have already assigned white. 
If the user wants to select another default color all original white cues will also become default colored. 
I am OK with using a different shade of white. We can also use the alpha channel to distinguish this white from the palette white. 
I have originally proposed orange, which also looks good and is the default color of record box.

Reply:
OK. @Be-ing has also commented on that. I'll start a discussion on Zulip.

Reply:
https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/.22No.20Color.22.20Replacement